### ISSUE TITLE
Arabic Version

### ISSUE THREAD

**Issue by mariokirolos** [martijnlentink/propresenter-custom-bibles#28]
First of all I can't thank you enough for such great tool, 
We have downloaded the arabic version of the bible a while ago and it's really working perfeclty fine, just 1 thing that I am not sure that it would be a bible related or importing related, 

As you know Arabic is Right to left, also Arabic has it's own numbers. While displaying the bible, it shows the numbers in the English format, as well it shows the numbers on the left while the numbers should be on the right, other than that it's perfect, 

I have added screenshot for your reference. Appreicate your response in advance :) 
![Screenshot 2024-07-20 at 9 17 12 AM](https://github.com/user-attachments/assets/1dba96da-7d0b-41dc-abca-798580b09f2e)

**Comment by martijnlentink** [comment]
Hi @mariokirolos,

Great to hear that the script is working out for you! It is good that you address this since I myself have zero experience with RTL languages. I do know that it is possible to provide script direction to the metadata of the bible but I don't think ProPresenter is taking that into account. I've taken a look at Iranian Persian (which is provided as a free bible) which is also a RTL language. Here I observed similar behaviour as you see with the arabic.

![image](https://github.com/user-attachments/assets/43c759d8-d403-4f22-8cb1-de49ce5192a1)

Looking at the metadata of the bible it looks like the script direction is correctly set to RTL.

![image](https://github.com/user-attachments/assets/5af068a8-a5e3-4349-b4ae-24af95b21573)

Now I got really curious and I decompiled the `BibleTemplateHelper`. Here I found that script direction is not taken into account in any way. The number is just always prepended.

![image](https://github.com/user-attachments/assets/0a752cc8-124e-49da-9e6c-f1ae0503bf4a)

So that sums it up basically. You'll need to create a ticket at RenewedVision asking if they can take translation of numbers and script direction into consideration when generating the bible slides.

TLDR;
Script direction is not set using our script, that needs to change. But looking at the source ProPresenter itself does not consider text direction when generating the slides. Conclusion: must be fixed first at ProPresenter end before it to work.

**Comment by mariokirolos** [comment]
Hi @martijnlentink 
Thank you so much for going the extra mile for me and trying different solutions. That's really appreciated. 

Sure thing, I will open a ticket with RenewedVision themselves.
